src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -67,7 +67,12 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 		// HTML representation of the output.
 		for (const mimeType of Object.keys(output.data)) {
 			if (mimeType === 'text/html') {
-				return this.createRawHtmlOutput(output.id, runtime, output.data[mimeType]);
+				return this.createRawHtmlOutput({
+					id: output.id,
+					runtimeOrSessionId: runtime,
```

Comment:
Nit: Given that the runtime is currently only used for its `sessionId`, I wonder if we could make the arg just `sessionId`?

Reply:
In the creation of the `WebviewInitInfo` there's a part that the extension ID is typed to be an `ExtensionIdentifier` and passing a string to it doesn't make typescript happy. https://github.com/posit-dev/positron/blob/836dc10789de912dd3c35a853813baeb3b3091d5/src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts#L283-L284